server/paymentProviders/stripe/creditcard.js

```diff
@@ -272,6 +274,8 @@ export default {
       if (error.message !== 'Payment Intent require action') {
         logger.error(`Stripe Payment Error: ${error.message}`);
         logger.error(error);
+        logger.error(error.stack);
```

Comment:
Isn't the stack already logged with the line before?

Reply:
It appears not: https://my.papertrailapp.com/systems/opencollective-prod-api/events?focus=1162302319234482181&q=PaymentIntent

Reply:
Ok, we can try like that.